stock_barcodes/wizard/stock_barcodes_read_picking.py

```diff
@@ -40,6 +43,31 @@ class WizStockBarcodesReadPicking(models.TransientModel):
         string='Confirmed moves',
     )
 
+    @api.depends(
+        "picking_id", "barcode", "picking_id.move_lines.move_line_ids.qty_done"
+    )
+    def _compute_pending_move(self):
+        for record in self:
+            text = ""
+            if record.picking_id:
+                moves = record.picking_id.move_ids_without_package.filtered(
+                    lambda r: r.product_uom_qty > r.quantity_done
+                )
+                if moves:
+                    text = self.env["ir.qweb"].render(
+                        "stock_barcodes.missing_moves", {
+                            "picking": record.picking_id,
+                            "moves": moves
+                        },
+                    )
+                else:
+                    text = self.env["ir.qweb"].render(
+                        "stock_barcodes.no_missing_moves", {
+                            "picking": record.picking_id,
+                        },
+                    )
```

Comment:
I think we can also improve code here and keep only one template stock_barcodes.missing_moves and at template level we can check if moves are there show the table else show the the message No pending operation ? What do your think ? @albariera